pynecone/components/tags/tag.py

```diff
@@ -62,20 +63,21 @@ def format_prop(
             if not prop.is_local or prop.is_string:
                 return str(prop)
             if issubclass(prop.type_, str):
-                return json.dumps(prop.full_name)
+                return json.dumps(prop.full_name, ensure_ascii=False)
```

Comment:
Might be worth it to add a helper function in `utils.py` so we never forget to add this.

Reply:
Ok，I ran the linter as the doc said and  commit again , please review the code.
And by the way , your guys did a fantastic job, I love this project and wish it goes well!